lib/services/auth.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:helping_hands/models/user.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+class AuthService {
+
+  final FirebaseAuth _auth = FirebaseAuth.instance;
+
+  // create user obj based on firebase user
+  User _userFromFirebaseUser(FirebaseUser user) {
+    return user != null ? User(uid: user.uid) : null;
+  }
+
+  // auth change user stream
+  Stream<User> get user {
+    return _auth.onAuthStateChanged
+      //.map((FirebaseUser user) => _userFromFirebaseUser(user));
+      .map(_userFromFirebaseUser);
+  }
+
+  // sign in anon
+  Future signInAnon() async {
+    try {
+      AuthResult result = await _auth.signInAnonymously();
+      FirebaseUser user = result.user;
+      return _userFromFirebaseUser(user);
+    } catch (e) {
+      print(e.toString());
```

Comment:
Does flutter have a logging service we can use, instead of just printing error messages?